sand/controllers/search.py

```diff
@@ -26,7 +26,7 @@ def get_search(name: Literal['classes', 'entities', 'props']) -> Union[IEntitySe
         GetSearchCache.search = {}
         search_config = SETTINGS["search"]
         constructor = search_config[name]
-        GetSearchCache.search[name] = import_func(constructor)()
+        GetSearchCache.search[name] = import_func(constructor)().extended_wikidata_search()
```

Comment:
should not do this.

Reply:
should I take this as a separate variable ?